android/build.gradle

```diff
@@ -27,6 +27,11 @@ android {
         archivesBaseName = "Unciv"
     }
 
+    // necessary for Android Work lib
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
Most phones don't support java 8, which is why the compilation to older JVM code is critical, more so than turn notification.
Is there an alternative to Android Work?

Reply:
Are you sure it's really a problem?
I just tried it in Android 15 with the emulator and had no problem despite the compile option.
Google also claims compatibility with [Android 14 for the lib,](https://developer.android.com/topic/libraries/architecture/workmanager) despite [requiring Java 8 for it.](https://developer.android.com/jetpack/androidx/releases/work#2.1.0)

Reply:
I'm not sure, and reading Google's documentation only made me more confused about this issue
So we'll try this out and worst case scenario is we get angry emails and then we can decide what to revert it :)

Reply:
Yeah, if there's a problem we can always downgrade to Android Work 2.0.1, the last version before the switch to Java 8.